Author: Robert James Clay <jame@rocasa.us>
Description: Set working directory in the required ledgersmb-starman.service file
 as well as the optional ledgersmb-plack-fcgi.service, ledgersmb_plackup.service,
 and ledgersmb-development_plackup.service files.
Origin: vendor
Forwarded: not-needed
Last-Update: 2018-10-09

--- a/doc/conf/systemd/ledgersmb-development_plackup.service
+++ b/doc/conf/systemd/ledgersmb-development_plackup.service
@@ -7,7 +7,7 @@
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb
 
 #If you do not want a log file, comment this and remove the access_log option below
 Environment=LOG=/var/log/ledgersmb/ledgersmb-development_plackup.log
@@ -15,6 +15,7 @@
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
 #Environment=PERL5LIB=/path/to/local-lib/lib/perl5
+Environment=LSMB_CONFIG_FILE=/etc/ledgersmb/ledgersmb.conf
 
 # Be sure to set a user and group below
 # which don't have write access to the directories
--- a/doc/conf/systemd/ledgersmb_plack-fcgi.service
+++ b/doc/conf/systemd/ledgersmb_plack-fcgi.service
@@ -7,12 +7,13 @@
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
 # to the correct location by uncommenting the line below
 #Environment=PERL5LIB=/path/to/local-lib/lib/perl5
+Environment=LSMB_CONFIG_FILE=/etc/ledgersmb/ledgersmb.conf
 Environment=PLACK_ENV=deployment
 
 # Be sure to set a user and group below
--- a/doc/conf/systemd/ledgersmb_plackup.service
+++ b/doc/conf/systemd/ledgersmb_plackup.service
@@ -7,9 +7,10 @@
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb
 
 #If you do not want a log file, comment this and remove the access_log option below
+Environment=LSMB_CONFIG_FILE=/etc/ledgersmb/ledgersmb.conf
 Environment=LOG=/var/log/ledgersmb/ledgersmb_plackup.log
 
 # In case you installed dependencies into a 'local::lib'
--- a/doc/conf/systemd/ledgersmb_starman.service
+++ b/doc/conf/systemd/ledgersmb_starman.service
@@ -7,12 +7,13 @@
 After=network.target
 
 [Service]
-WorkingDirectory=WORKING_DIR
+WorkingDirectory=/usr/share/ledgersmb
 
 # In case you installed dependencies into a 'local::lib'
 # make sure you set the PERL5LIB environment variable
 # to the correct location by uncommenting the line below
 #Environment=PERL5LIB=/path/to/local-lib/lib/perl5
+Environment=LSMB_CONFIG_FILE=/etc/ledgersmb/ledgersmb.conf
 
 # Be sure to set a user and group below
 # which don't have write access to the directories
